### PR TITLE
Add record button to toolbar view.

### PR DESCRIPTION
**Pull request by kannanvijayan**
It's not in the exact right place yet, but it's circular, red, and prints some stuff to stderr when it's clicked.

### CODE REVIEW

**Comment by markerikson** [comment]
This is a stupid amount of code for one lousy button :)

**Comment by kannanvijayan** [comment]
This is our interactive recording MVP.  We:
1. Show a record button in the chromium UI
  - But not when `RECORD_ALL_CONTENT` envar is set.
2. When record button is clicked, it opens a new tab to the current active URL
  - The new tab records itself.

Verified that only one recording is created when chromium is used this way (all processes are not recorded).

Not implemented:
* Record button should change to a "stop" button when at a tab that is currently recording.

File: content/browser/renderer_host/render_process_host_impl.cc [hunk 1]
```diff
@@ -1423,6 +1424,7 @@ int RenderProcessHost::GetCurrentRenderProcessCountForTesting() {
 RenderProcessHost* RenderProcessHostImpl::CreateRenderProcessHost(
     BrowserContext* browser_context,
     SiteInstanceImpl* site_instance) {
+
```
Comment (Domiii) [inline content/browser/renderer_host/render_process_host_impl.cc:1427]: Unwanted line?

File: content/browser/renderer_host/render_process_host_impl.cc [hunk 2]
```diff
@@ -3205,6 +3214,9 @@ void RenderProcessHostImpl::AppendRendererCommandLine(
     command_line->AppendSwitchASCII(blink::switches::kJavaScriptFlags,
                                     "--jitless");
 
+  // [RecordReplay] Call `IsForRecordReplayRecording()` here and
```
Comment (Domiii) [inline content/browser/renderer_host/render_process_host_impl.cc:3217]: Is this a left-over comment?

**Review by Domiii — APPROVED** [review]

File: content/browser/renderer_host/render_process_host_impl.cc (hunk 2 above)
Comment (kannanvijayan) [reply content/browser/renderer_host/render_process_host_impl.cc:3217]: Likely.  I thought I had gotten all of them, will check.

**Comment by kannanvijayan** [comment]
This looks good on CI: https://buildkite.com/replay/testing-runtime-e2e/builds/1770
Just waiting for windows tests to finish at the moment, everything else is green.  Landing.

**Pull request merged**